.github/workflows/ci-openapi.yml

```diff
@@ -1,17 +1,11 @@
-# Update database ERD diagrams and OpenAPI docs so that they remain up to date with the application
 name: Update Database ERD Diagrams and OpenAPI Docs
 
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        required: true
-        default: "main"
-        description: Tag or branch or SHA to deploy
-
-defaults:
-  run:
-    working-directory: ./api
+  pull_request:
+    paths:
+      - api/**
+      - api/Makefile
+      - .github/workflows/ci-openapi.yml
```

Comment:
I actually really don't want this back in our PRs, it causes constant issues.

* A person sends a PR out, that PR has failures.
* There was a change to the schema that gets created and committed to a PR. The tests/linting don't run.
* GitHub sees that the latest change on a PR are fine and has no issues with merging it.

It also just doesn't work for anyone who has forked the repo as it can't commit to their fork properly.

Is there a way we can make this a daily/weekly job that generates a PR for us to approve/merge?